test/integration/scenarios/03.basicWithNsDelete/magefile.go

```diff
@@ -82,7 +82,13 @@ func Test() error {
 // Run this manually to prepare your local environment for testing/debugging
 func Setup() error {
 	testhelpers.SetupCluster()
-
+	err := shellcmd.Command(fmt.Sprintf("kubectl get secret container")).Run()
+	if err != nil {
+		err := shellcmd.Command(fmt.Sprintf("kubectl create secret generic container --from-literal=port=%d", 9090)).Run()
+		if err != nil {
+			return err
+		}
+	}
```

Comment:
I would prefer applying a secret yaml file to create the secret and it will make the test case more clearly.